### DISCUSSION TITLE
Video

### DISCUSSION THREAD

**Discussion by nickel-dime** [ValentinH/react-easy-crop#349] [Q&A]
How would you add support for uploading a video and being able to crop it? When using the link "https://vid.ly/5u4h3econtent=video" my cropper works fine, but when I send my own uploaded MP4 it does not. 
I am currently using the FileReader class, and it sends the video but there is no grid and width/height, just an image played which I can zoom in on. Any ideas would be amazing!

<img width="584" alt="Screen Shot 2022-02-11 at 1 04 12 PM" src="https://user-images.githubusercontent.com/55257208/153645136-ba5d7d6c-1079-4edb-aaea-997bf9117255.png">

**Comment by ValentinH** [comment]
I have never worked with videos, but I don't think you can convert it to data url. This is why you are seeing an image.

You probably need to upload somewhere and get a public URL to display it on the page.
